tgui/packages/tgui/interfaces/PreferencesMenu/CharacterPreferences/names.tsx

```diff
@@ -154,7 +154,7 @@ export function NameInput(props: NameInputProps) {
             <Input
               autoSelect
               expensive
-              onChange={updateName}
+              onEnter={updateName}
```

Comment:
As per discord, this should be `onBlur` for ux reasons- and cause that was the previous behaviour
```suggestion
              onBlur={updateName}
```